optimum/utils/normalized_config.py

```diff
@@ -101,6 +101,23 @@ def __getattr__(self, attr_name):
         return super().__getattr__(attr_name)
 
 
+class NormalizedEncoderDecoderConfig(NormalizedConfig):
+    ENCODER_NORMALIZED_CONFIG_CLASS = None
+    DECODER_NORMALIZED_CONFIG_CLASS = None
+
+    def __getattr__(self, attr_name):
+        if self.ENCODER_NORMALIZED_CONFIG_CLASS is not None and attr_name.upper() in dir(
+            self.ENCODER_NORMALIZED_CONFIG_CLASS
+        ):
+            return self.ENCODER_NORMALIZED_CONFIG_CLASS.__getattr__(attr_name)
+        if self.DECODER_NORMALIZED_CONFIG_CLASS is not None and attr_name.upper() in dir(
+            self.DECODER_NORMALIZED_CONFIG_CLASS
+        ):
+            return self.DECODER_NORMALIZED_CONFIG_CLASS.__getattr__(attr_name)
+
+        return super().__getattr__(attr_name)
```

Comment:
Couldn't the `attr_name` be in both the encoder and decoder config? e.g. `bos_token_id` in https://huggingface.co/microsoft/trocr-small-handwritten/blob/main/config.json . Could this result in unexpected behavior? We should rather be able to access attributes recursively no, like `normalized_config.encoder.bos_token_id` or `normalized_config.decoder.bos_token_id`, if this makes sense!

Reply:
Yes, it could be possible. This seems to be a common problem for the other hybrid configs also. Would make a note  of it and handle it separately. 

Currently, this would not affect the ORT export (where we export the encoder and decoder separately) as one of the configs would always be None. So considering it low priority.